test/tests/mfem/kernels/tests

```diff
@@ -199,6 +210,8 @@
     requirement = 'The system shall have the ability to solve for an irrotational vortex in two dimensions using MFEM.'
     abs_zero = 1e-6
     capabilities = 'mfem'
+    devices = 'cpu cuda'
     recover = false
+    max_parallel = 1 # mfem-hypre doesn't appear to work with multiple ranks with cuda
```

Comment:
Let's rebase so we don't miss any tests. Given pretty much all tests require `max_parallel = 1`, I wonder if we should move this comment up to the beginning of the file or somewhere else.

Reply:
We have a bit of interweaving issues. The `XMLDiffs` innately can't be used with more than one rank based on conversations with @alexanderianblair. And then we have this MOOSE-MFEM-HypreGMRES issue

Reply:
> We have a bit of interweaving issues. The `XMLDiffs` innately can't be used with more than one rank based on conversations with @alexanderianblair. And then we have this MOOSE-MFEM-HypreGMRES issue

This is true, since the partitioning may differ between runs, and the number of output files in a `ParaViewDataCollection` will depend on the number of processors used.

We have a couple of `VectorPostProcessors` inbound for sampling MFEM FE variables at arbitrary points using  GSLib from @k-collie - which should enable us to convert many of the existing XMLDiff tests over to CSVDiff tests (also significantly reducing output sizes...) - WIP branch is [here](https://github.com/aurora-multiphysics/platypus-moose/tree/k-collie/mfem-point-and-line-sampler)